src/main/java/lt/swedbank/itacademy/ItAkaLeasingSystemBackEnd/services/CustomerService.java

```
@@ -0,0 +1,121 @@
+package lt.swedbank.itacademy.ItAkaLeasingSystemBackEnd.services;
+
+import lt.swedbank.itacademy.ItAkaLeasingSystemBackEnd.beans.documents.BusinessCustomer;
+import lt.swedbank.itacademy.ItAkaLeasingSystemBackEnd.beans.documents.Customer;
+import lt.swedbank.itacademy.ItAkaLeasingSystemBackEnd.beans.documents.PrivateCustomer;
+import lt.swedbank.itacademy.ItAkaLeasingSystemBackEnd.beans.enums.CustomerType;
+import lt.swedbank.itacademy.ItAkaLeasingSystemBackEnd.beans.response.BusinessCustomerResponse;
+import lt.swedbank.itacademy.ItAkaLeasingSystemBackEnd.beans.response.CustomerResponse;
+import lt.swedbank.itacademy.ItAkaLeasingSystemBackEnd.beans.response.PrivateCustomerResponse;
+import lt.swedbank.itacademy.ItAkaLeasingSystemBackEnd.repositories.CustomerRepository;
+import lt.swedbank.itacademy.ItAkaLeasingSystemBackEnd.utils.PasswordEncryption;
+import lt.swedbank.itacademy.ItAkaLeasingSystemBackEnd.utils.UserIDGenerator;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import javax.validation.Valid;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Lukas
+ */
+@Service
+public class CustomerService{
+
+    @Autowired
+    private CustomerRepository customerRepository;
+
+    public boolean existsCustomerByUserID(String userID) {
+        return customerRepository.existsCustomerByUserID(userID);
+    }
+
+    public boolean existsCustomerByEmail(String email){
+        return customerRepository.existsCustomerByEmail(email);
+    }
+
+    public boolean existsCustomerByUserIDAndEmail(String userID, String email){
+        return customerRepository.existsCustomerByUserIDAndEmail(userID, email);
+    }
+
+    public Customer findCustomerByEmail(String email){
+        return customerRepository.findCustomerByEmail(email).orElse(null);
```

returning null isn't the best option. would be better to throw an exception